controller/src/main/java/io/pravega/controller/task/Stream/TaskStepsRetryHelper.java

```diff
@@ -20,8 +20,8 @@
 class TaskStepsRetryHelper {
     private static final long RETRY_INITIAL_DELAY = 100;
     private static final int RETRY_MULTIPLIER = 2;
-    private static final int RETRY_MAX_ATTEMPTS = 100;
-    private static final long RETRY_MAX_DELAY = Duration.ofSeconds(5).toMillis();
+    private static final int RETRY_MAX_ATTEMPTS = 10;
```

Comment:
The caller of the task should retry "indefinitely" or multiple times depending on their need. 
the task itself should typically complete with success or failure soon enough. 
thats why having high retry count for task steps may make the task linger for longer.